components/global-navigation.md

```diff
@@ -6,12 +6,8 @@
 	<div class="wds-global-navigation">
 		<div class="wds-global-navigation__content-bar">
 			<a href="#" class="wds-global-navigation__logo">
-				<svg class="wds-global-navigation__logo-fandom" alt="Fandom">
-					<use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#wds-company-logo-fandom"></use>
-				</svg>
-				<span class="wds-global-navigation__logo-powered-by">powered by</span>
-				<svg class="wds-global-navigation__logo-wikia" alt="Wikia">
-					<use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#wds-company-logo-wikia"></use>
+				<svg class="wds-global-navigation__logo-fandom" alt="Fandom powered by Wikia">
```

Comment:
`wds-global-navigation__logo-fandom-powered-by-wikia`? Not sure if it changes anything.